tests/regression_tests.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 14 16:17:57 2020
+
+@author: smullally
+"""
+
+
+import exovetter as vet
+
+
+def test_lpp():
+    """
+    Test reads in an lpp file name, runs the metric without \
+    any plots and returns values.  
+    It then asseerts that the normal one is greater than 0, which must be true.
+    """
+    
+    tce_filename = "tce0001234567_dvt.fits"
+    
+    #Populate the tce class by reading in a file
+    #At this point a phases array, 
+    #and (int) transit_number would get populated.
+   
+    tce = vet.create_tce(file = tce_filename, ext = 1)
```

Comment:
This is more of what I was thinking off when we were discussing the API yesterday. I was under the impression that there would be a class and reading the file is a *class* method (not instance method).
```suggestion
    tce = vet.TCEClass.from_file(tce_filename, ext=1)
```

Reply:
Ah yes, that is better.

I'm also thinking we need to rethink what we mean by the tce and separate out the light curve information. But that opens up other issues.  So I need to rethink a bit.